### PR TITLE
Provide a BC layer for files in "app/config/" referenced by PluginSkeleton

### PR DESCRIPTION
**Pull request by pamil**
Files `app/config/routing.yml`, `app/config/routing_dev.yml` and `app/config/security.yml` are referenced in PluginSkeleton. This PR provides a backwards compatibility layer for them.

### CODE REVIEW

File: app/config/parameters.yml.dist [hunk 1]
```diff
@@ -1,24 +0,0 @@
-# This file is auto-generated during the composer install
```
Comment (pamil) [inline app/config/parameters.yml.dist]: This file is not referenced by neither Sylius-Standard nor PluginSkeleton.

File: app/config/_container_deprecation.php [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+ * file that was distributed with this source code.
+ */
+
+/**
+ * @internal
+ */
+
+declare(strict_types=1);
+
+@trigger_error('Importing files from Sylius/Sylius\'s "app/config" directory is deprecated since Sylius 1.3.', \E_USER_DEPRECATED);
```
Comment (pamil) [inline app/config/_container_deprecation.php]: Deprecating YAML files is kinda weird 🎉

File: app/config/parameters.yml.dist (hunk 1 above)
Comment (pamil) [reply app/config/parameters.yml.dist]: Ah, we still need it here, it is being removed in #9665.

**Comment by lchrusciel** [comment]
Thanks Kamil!

**Comment by pamil** [comment]
@teohhanhui I'm trying to solve that by making Sylius-Standard a part of Sylius/Sylius (which is subtree split back into Sylius-Standard). This way we can run tests on this repo while having one less place with configuration.

**Comment by teohhanhui** [comment]
Yes, make them part of the application, not part of Sylius. Sylius-Standard should be treated purely as a distribution, aka recommended scaffolding / project template. No BC guarantees.

**Pull request merged**